src/components/charts/AddVisualizationForm.js

```diff
@@ -54,22 +75,27 @@ const AddVisualizationForm = ({
             label={t('Dataset')}
             className={classes.select}
           >
-            {datasets.map(({ id, name }) => (
+            {datasets.map(({ id, name, schemaType }) => (
               <MenuItem value={id} key={id}>
                 {name}
+                {schemaType && schemaType !== SCHEMA_TYPES.NONE && (
+                  <SchemaTag
+                    schemaType={schemaType}
+                    className={classes.schemaTag}
+                  />
+                )}
               </MenuItem>
             ))}
           </Select>
         </FormControl>
         <div>
-          <Button
-            variant="contained"
-            color="primary"
-            disabled={!selectedDatasetId}
-            onClick={fetchDatasetToBeVisualized}
-          >
-            {t('Visualize')}
-          </Button>
+          {!selectedDatasetId || isGraasp ? (
+            button
+          ) : (
+            <Tooltip title={t('Only Graasp datasets can be visualized')}>
+              <div>{button}</div>
```

Comment:
Is the `div` necessary?

Reply:
Yes, otherwise the tooltip doesn't show on disabled buttons